src/journal/test-journal-send.c

```diff
@@ -36,7 +40,7 @@ int main(int argc, char *argv[]) {
 
         assert_se(sd_journal_perror("") == 0);
 
-        memset(huge, 'x', sizeof(huge));
+        memset(huge, 'x', HUGE_SIZE);
         memcpy(huge, "HUGE=", 5);
         char_array_0(huge);
```

Comment:
Ooops, char_array_0 cannot be used on a pointer.

Reply:
Thanks, updated. There's another issue in `test_lz4_decompress_partial()` where the string was not terminated.